init.zsh

```diff
@@ -17,6 +17,29 @@ if ! autoload -Uz is-at-least || ! is-at-least "$min_zsh_version"; then
 fi
 unset min_zsh_version
 
+# zprezto convenience updater
+function zprezto-update {
+    local OWD="$(pwd)"
+    cd -- "${ZPREZTODIR}"
+    local orig_branch="$(git symbolic-ref HEAD 2>/dev/null | cut -d '/' -f 3)"
+    if [[ "$orig_branch" == "master" ]]; then
+        git fetch
+        if [[ "$(git status -uno | grep behind)" ]]; then
```

Comment:
This check seems a bit error prone. Maybe something like what's mentioned here would be better: https://stackoverflow.com/a/3278427 (on a side note, I had no idea `@` even existed in git)